spec/system/visit_msa_component_new_spec.rb

```diff
@@ -40,6 +41,52 @@
     end
   end
 
+  context 'creation fails without unique entity id' do
+    it 'when entity id already exists for a msa component' do
+      existing_component = msa_component
+      entity = 'http://test-entity-id'
+      component_name = 'test component'
+      visit new_msa_component_path
+      fill_in 'component_name', with: component_name
+      fill_in 'component_entity_id', with: entity
+      select team.name, from: "component_team_id"
+      choose('component_environment_staging')
+      click_button 'Create MSA component'
+
+      expect(page).to have_content t('components.errors.existing_entity_id')
+    end
+  end
+
+  context 'creation is successful with only leading and trailing whitespaces on entity id' do
+    it 'when entity id in a msa component has leading and trailing whitespaces' do
```

Comment:
Can this test validate that the entity ID of the newly created component is `'http://test-entity-id-2'`? I.e. verify that whitespaces have been stripped